lib/msal-react-native-android-poc/MSALModule/android/src/main/java/com/reactlibrary/MSALModule.java

```diff
@@ -89,6 +97,47 @@ public void onCancel() {
         };
     }
 
+     /**
+     * getAccount(): retrieves account currently signed in
+     * Parameters: Promise promise (resolve will return the account as a map if it exists; reject will return null for nonexistent map or error)
+     */
+    @ReactMethod
+    public void getAccount(Promise promise) {
+        //first load account and get status string
+        String message = loadAccount();
+        if (mAccount == null) {
+            promise.reject("loadaccountnull", message);
+        } else {
+            promise.resolve(mapAccount(mAccount));
+        }
+    }
+
+    /**
+     * loadAccount(): will load a currently signed in account. Called in constructor.
+     * Returns a String with a message of the status (possibly an exception)
+     */
+
+    private String loadAccount() {
+        if (publicClientApplication == null) {
+            return "publicClientApplication is null";
+        }
+        String message = "";
+        try {
+            ICurrentAccountResult result = publicClientApplication.getCurrentAccount();
+            IAccount currAccount = result.getCurrentAccount();
+            if (currAccount == null) {
+                message = "No account currently signed in";
+            } else {
+                mAccount = currAccount;
+                message = "Retrieved signed in account";
```

Comment:
This message variable looks like it isn't being used unless an error occurs. Is that right? Consider removing this variable and instead add verbose logging here to help with debugging

Reply:
Initially the message variable was for logging in the react console (currently there isn't really a way to log to the react console directly from the android module) but my implementation wasn't the best. I'll focus on logging in the android console to be consistent.